README.md

```diff
@@ -11,6 +11,8 @@ Simple node module for downloading your precious [Tumblr](https://tumblr.com) li
 A few things for having in mind while using it:
 * This will save ***just image files*** and not any other kind of content.
 * The number of `postsToLoad` it likely ***NOT*** going to be the same as the number of images that you will get. Posts can have more than one image or not having any image ***AT ALL***.
+* The maximum value for postsOffset is 1000.
```

Comment:
Can you please add [this](https://www.tumblr.com/docs/en/api/v2#blog-likes) url here to let the users know that this maximum value is coming from the Tumblr API?